site/src/_pages/docs/4_faq.mdx

```diff
@@ -244,9 +244,19 @@ All schemas created on the Block Protocol website are currently public. In futur
-1.  The [schema.org](https://schema.org) ontology is slow to change. This is by design, similar to how most standards organizations operate. And while it can move fast when clear universal impetus exists (evidenced by its excellent response to COVID) it ultimately represents a pseudo-centralized model of maintaining a schema registry. The Block Protocol provides the permissionless ability to build atop and extend [schema.org](https://schema.org), unbounded by the processing constraints and imagination of any single working group or organization.
-1.  Not everything is included in the [schema.org](https://schema.org) ontology. For example, although _animal shelters_ and _pet stores_ exist, _animals_ and _pets_ themselves are missing entirely. This arguably reflects the priorities of schema.org’s maintainers (notably corporates, and search engines at that).
+1.  On other occasions you’ll want to store information differently than how it’s set out in the [Schema.org](http://schema.org) definition. For example, [schema.org/Person](http://schema.org/Person) defines a person as having a `givenName` and a `familyName` and yet in various cultures this isn’t guaranteed. HASH uses `preferredName` and `legalName` instead - in communications and billing contexts respectively. You can view the HASH ‘Person’ schema at [https://blockprotocol.org/@hash/types/Person](/@hash/types/Person) and see how it crosswalks with the canonical definition of a person provided by Schema.org. This is just one example of how custom schemas can be made to relate back to the core ontology provided by Schema.org.
+1.  The [Schema.org](https://schema.org) ontology is slow to change. This is by design, similar to how most standards organizations operate. And while it can move fast when clear universal impetus exists (evidenced by its excellent response to COVID) it ultimately represents a pseudo-centralized model of maintaining a schema registry. The Block Protocol provides the permissionless ability to build atop and extend [Schema.org](https://schema.org), unbounded by the processing constraints and imagination of any single working group or organization.
+1.  Not everything is included in the [Schema.org](https://schema.org) ontology. For example, although _animal shelters_ and _pet stores_ exist, _animals_ and _pets_ themselves are missing entirely. This arguably reflects the priorities of schema.org’s maintainers (notably corporates, and search engines at that).
+
+</FAQ>
+
+<FAQ question="Doesn't a decentralized schema registry result in problems?">
+
+Moving beyond a single set of constrained schemas that rarely change introduces new issues. We believe that all of these issues can be mitigated and that the benefits of an open ecosystem of well-described, discoverable types grealy outweigh the drawbacks.
```

Comment:
```suggestion
Moving beyond a single set of constrained schemas that rarely change introduces new issues. We believe that all of these issues can be mitigated and that the benefits of an open ecosystem of well-described, discoverable types greatly outweigh the drawbacks.
```

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/pull/624/commits/3e19942e1b1d5685cd9d6861b0d9907e0ce1c854